src/outdated-browser-message.js

```diff
@@ -0,0 +1,30 @@
+import 'outdated-browser-rework/dist/style.css';
```

Comment:
This style, while it does look like Terra, is extreme. I'd like to ensure we get PM feedback/approval before merging.